tests/sync/test_sync.py

```diff
@@ -43,7 +43,10 @@ def test_sync_query_selector(page: Page) -> None:
 
 def test_page_repr(page: Page) -> None:
     page.goto("https://example.com")
-    assert repr(page) == f"<Page url={page.url!r}>"
+    assert (
+        repr(page)
+        == "<Page url='https://example.com/'> frame=<Frame name= url='https://example.com/'>"
```

Comment:
In my opinion is the page's main frame name always empty, thats why I think we don't have to add the frame to the Page repr.

Reply:
If it's always empty then why does that even exists ?

Reply:
Only sub-frames of a page can have named frames. The main frame's name is always an empty string.